.travis.yml

```diff
@@ -1,18 +1,26 @@
-language: python
+language: c
 
-python:
-    - 2.7
+os:
+    - osx
+    - linux
 
-env: DISPLAY=:99 MOZ_XVFB=1
+compiler:
+    - gcc
```

Comment:
Nothing in our build needs gcc.

For Travis multi-os for our Python tests, we need to use one of Travis's C workers, and if you don't specify any compiler, it will create builds for both gcc and clang, for both Linux and OS X. So I added the compiler specification simply to restrict our builds to one each for Linux and OS X.